google-java-format.gradle

```diff
@@ -0,0 +1,71 @@
+/**
+ * Reformat the diff from a given commit, formatBase. By default formatBase=HEAD.
+ * Requires that bash can run executable python 2.7 files directly.
+ *
+ * Usage:
+ * ./gradlew googleJavaFormat [-PformatBase=<format base>]
+ * 
+ * To enable this task when build, explicitly specify property "formatBase"
```

Comment:
Don't abuse the `formatBase` configuration for this. Use a separate property, like "formatJavaOnBuild".

You might also want to make it clear you are referring to the `build` _task_.